plugins/clnrest/utilities/rpc_plugin.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+from multiprocessing import Manager
+from pyln.client import Plugin
+
+plugin = Plugin(autopatch=False)
+manager = Manager()
+queue = manager.Queue()
+
+plugin.add_option(name="rest-certs", default=os.getcwd(), description="Path for certificates (for https)", opt_type="string", deprecated=False)
+plugin.add_option(name="rest-protocol", default="https", description="REST server protocol", opt_type="string", deprecated=False)
+plugin.add_option(name="rest-host", default="127.0.0.1", description="REST server host", opt_type="string", deprecated=False)
+plugin.add_option(name="rest-port", default=3010, description="REST server port to listen", opt_type="int", deprecated=False)
+
+def add_notifications(event, message):
+    queue.put(str({"event": event, "notification": str(message)}) + "\n")
```

Comment:
from the python doc

```
Put obj into the queue. If the optional argument block is True (the default) and timeout is None (the default), block if necessary until a free slot is available. If timeout is a positive number, it blocks at most timeout seconds and raises the [queue.Full](https://docs.python.org/3/library/queue.html#queue.Full) exception if no free slot was available within that time. Otherwise (block is False), put an item on the queue if a free slot is immediately available, else raise the [queue.Full](https://docs.python.org/3/library/queue.html#queue.Full) exception (timeout is ignored in that case).
```

So this will throw an exception when there are too many notifications, I guess this is the common case

Reply:
@vincenzopalazzo Fair point. Shouldn't be the problem after ws.